charts/llmos-agents/templates/langflow/frontend-nginx-configmap.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "llmos-agents.langflow.fullname" . }}-frontend-nginx-config
+  labels:
+    {{- include "llmos-agents.langflow.labels" . | nindent 4 }}
+    langflow-scope: "frontend"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)